tests/nosetests/pyanaconda_tests/module_network_test.py

```diff
@@ -56,6 +56,14 @@ def get_current_hostname_test(self):
         """Test getting current hostname does not fail."""
         self.network_interface.GetCurrentHostname()
 
+    def connected_test(self):
+        """Test getting connectivity status does not fail."""
+        self.network_interface.Connected
+
+    def connecting_test(self):
+        """Test checking connecting status does not fail."""
+        self.network_interface.IsConnecting()
+
```

Comment:
You can probably extend the tests a little if you mock the `nm_client`.

Reply:
I can't figure out how to mock nm_client reasonbly with current code, I've only added one silly test. In the future when I start to use the client heavily for real work with connections and ifcfgs, I'll probably encapsulate it into a separate module so maybe then it will be easier. For now the client (and NM / libnm) seems to be too bound to the network module.

Reply:
Added a test with mocked NMClient